packages/playground/src/utils/validators.ts

```diff
@@ -1,9 +1,12 @@
+import { default as StellarSdk, StrKey } from "stellar-sdk";
```

Comment:
```suggestion
import StellarSdk from "stellar-sdk";
```

we don't use `StrKey` here